rust/theoros/src/types/starknet_rpc.rs

```diff
@@ -0,0 +1,44 @@
+use anyhow::Result;
+use pragma_utils::conversions::starknet::felt_vec_to_vec_string;
+use starknet::{
+    core::types::{BlockId, BlockTag, EthAddress, Felt, FunctionCall},
+    macros::selector,
+    providers::{jsonrpc::HttpTransport, JsonRpcClient, Provider},
+};
+use url::Url;
+
+use super::StorageLocation;
+
+pub struct StarknetRpc(JsonRpcClient<HttpTransport>);
+
+impl StarknetRpc {
```

Comment:
let's create an issue for using cainome later on, for safer interaction with contracts

Reply:
added here:
https://github.com/astraly-labs/pragma-monorepo/issues/22